### DISCUSSION TITLE
Daemonize vs. KDE

### DISCUSSION THREAD

**Discussion by wmf39** [borgbase/vorta#1972] [General]
Is "daemonize" working with KDE? With Plasma 5 and 6 I have the following problem:

- Checkbox "Automatically start Vorta at login" active
- Checkbox "Show main window of Vorta on launch" not active

After login Vorta always shows the GUI. So I set both checkboxes to inactive and created the autostart entry in system settings manually. But this entry is deleted when Vorta starts. If I remove "--deamonize" manually from the "vorta.desktop" file, it is recovered when Vorta starts.

I solved it by uncheck both checkboxes, created the autostart entry in system settings manually and renamed the "vorta.desktop" file to "vortastart.desktop". That prevents the file from getting deleted by Vorta and Vorta is starting with only the icon in tray as desired. 

Tumbleweed 20240314, Plasma 6.0.2, Vorta 0.9.1

Edit: I discovered, that the workaround works when log out => log in, but not when reboot. Seems to me that something brings the window to foreground after the "daemonize" flag was checked. Maybe the check should be done when everything is loaded and send window to tray if visible.

**Comment by wmf39** [comment]
Ok, cause I got it not to work native, but realized "daemonize" works in terminal on first call (on second the app comes up), I created the following script:

```
if [ "$(ps -e | egrep -i vorta)" == '' ]; then
    echo "Starting Vorta..."
    vorta --daemonize
else
    echo "Vorta already started!"
fi
exit 0;
```
Both checkboxes  inactive, copied script to _~/.config/plasma-workspace/env/_ and run by plasma autostart it works with log off/in and reboot. Don't know what's wrong with plasma autostart when used to start vorta native. ¯\\_(ツ)_/¯

**Comment by real-yfprojects** [comment]
Is this the same as #1762?

**Reply by wmf39** [reply]
Yes it is, thank you. I removed the script, enabled "Automatically start Vorta at login" and added vorta to the desktop session ignore list. That did the trick.
